ui/src/tool-panel/location-track/location-track-infobox.scss

```diff
@@ -69,30 +97,52 @@ $-color-warning: vayla-design.$color-lemon-dark;
 }
 
 .location-track-infobox__split-close-button {
-    margin-left: 5px;
-    border-radius: 18px;
-    width: 20px;
+    width: 22px;
     height: 20px;
-    fill: vayla-design.$color-white-default;
+    fill: vayla-design.$color-blue-dark;
     display: flex;
     justify-content: center;
     align-items: center;
 
     &--disabled {
-        background-color: vayla-design.$color-black-lighter;
+        fill: vayla-design.$color-black-lighter;
     }
 
     &--enabled {
-        background-color: vayla-design.$color-blue-default;
+        fill: vayla-design.$color-blue-dark;
         cursor: pointer;
 
         &:hover {
-            fill: vayla-design.$color-white-default;
-            background: vayla-design.$color-blue-light;
+            fill: vayla-design.$color-blue-light;
         }
     }
 }
 
+.location-track-infobox__split-switch-error-msg {
+    padding: 4px;
+    padding-left: 12px;
+    margin-left: 12px;
+
+    @include vayla-design.typography-caption;
+    color: vayla-design.$color-black-default;
+    background-color: vayla-design.$color-lemon-light;
+}
+
+.location-track-infobox__split-switch-error-msg--start {
+    margin-bottom: 4px;
+    margin-top: -4px;
```

Comment:
Nämä miinusmerkkiset marginit on vähän jänniä. Toisaalta jos menee helpolla näin ja ei mene kohtuuella ei-negatiivisilla margineilla, niin mennään sitten näillä.